server/src/controllers/authTeamController.js

```diff
@@ -0,0 +1,102 @@
+import rp from 'request-promise';
+
+const server = 'http://localhost:8080';
```

Comment:
any way to keep this set somewhere globally so you don't have to keep declaring it in every module?